### ISSUE TITLE
Status and plan for v6 LTS

### ISSUE THREAD

**Issue by Fishrock123** [nodejs/LTS#153]
cc @nodejs/lts we should come with an interim plan for v6 at least for now.

I worked on a script for the bot a lot last week which should help here but it's not quite "production-ready" yet I'd say: https://github.com/nodejs/github-bot/pull/90

**Comment by jasnell** [comment]
Plan for what precisely?

**Comment by TheAlphaNerd** [comment]
I was planning on doing a batch backport similar to what I've done for v4
in the past. having more people take on the responsibility would obviously
be awesome

**Comment by Fishrock123** [comment]
> Plan for what precisely?

Upcoming releases &/or process

**Comment by jasnell** [comment]
If we can get automated checking/validation of backportable semver-patch PRs working reliably then that would definitely make things easier. Based on a number of conversations I've had with folks, I'm thinking that one release per month is a solid cadence.

**Comment by mhdawson** [comment]
I'm +1 on the 1/release per month cadence.

**Comment by bnoordhuis** [comment]
That doesn't mean delaying security fixes for up to a month, right?  Only normal releases?

**Comment by evanlucas** [comment]
I'm +1 on 1/month with as many as needed for security releases

**Comment by Fishrock123** [comment]
@bnoordhuis correct.

**Comment by TheAlphaNerd** [comment]
So I'm doing my first bit of backporting of both v4 and v6 right now and finding that it surprisingly does not add _that much_ work. Since everything needs to be audited for both line it can be done in parallel. I'm still looking to get some help and I know that @gibfahn is interested in getting involve, but figured it would be good to report that I don't think managing 2x the LTS releases will be anywhere close to 2x the work

**Comment by TheAlphaNerd** [comment]
Discussed in today's wg meeting. Going to open another issue to lay out released for next couple month's
